### ISSUE TITLE
36-37 Mh/s - 1080Ti - What am I missing?

### ISSUE THREAD

**Issue by kryptomining** [OhGodACompany/OhGodAnETHlargementPill#79]
Hello,
One card is running slow, and I need some help from you experts :) I have two 1080Ti cards (different brands)

1. Running smooth 54-55 Mh/s @ PL70 | Core+100 | Mem+900 | 62 degrees
2. Slow! 36-37 Mh/s @ PL 65 | Core 0 | Mem+300 | 53 degrees

No matter what i try i cannot get it higher than 36-37, the powerdraw is around 140w, which is lower than the other card, which is 160-170w.
Here is what I tried:
*Starting the pill with or without RevA flag (including specifing the correct GPU for RevA)
*Any PL, Core or Mem changes lowers the hashrate

Thought about changing the powerstate, but "nvidia-smi -q -d SUPPORTED_CLOCKS | more" gives me no values for that GPU, says N/A.

Here are the two GPUs:
__________________________________________________________________________
Display device       : GeForce GTX 1080 Ti on GP102-A GPU
Display driver       : 398.11
BIOS                 : 86.02.39.00.22
GUID                 : VEN_10DE&DEV_1B06&SUBSYS_85E51043&REV_A1&BUS_1&DEV_0&FN_0
Multi-GPU role       : master
Registry key         : \Registry\Machine\System\CurrentControlSet\Control\Video\{1A5C3563-783F-11E8-8C63-708BCD9CE39E}\0000
VDDC controller      : Display driver

GPU2
________________________________________________________________________________
Display device       : GeForce GTX 1080 Ti on GP102-A GPU
Display driver       : 398.11
BIOS                 : 86.02.40.00.4d
GUID                 : VEN_10DE&DEV_1B06&SUBSYS_1B0610B0&REV_A1&BUS_8&DEV_0&FN_0
Multi-GPU role       : synchronized with master
Registry key         : \Registry\Machine\System\CurrentControlSet\Control\Video\{1A5C3424-783F-11E8-8C63-E11E7F265D0A}\0000
VDDC controller      : Display driver

Any help would be greatly appreciated!

**Comment by jimmykl** [comment]
Hey @kryptomining what make and model is the limp card?

**Comment by jimmykl** [comment]
Unfortunately from various [reports](https://github.com/OhGodACompany/OhGodAnETHlargementPill/issues/47) Asus Turbos seem to be immune to the pill.

**Comment by kryptomining** [comment]
Thank you, I did read that. However it seems it could reach above 40 or even 44 at least. Should I continue in the other thread?

**Comment by jimmykl** [comment]
Yeah, since it's related it could be good to add your experience there so it's all in one place

**Comment by kryptomining** [comment]
Will do, thanks

**Comment by bracinac** [comment]
I have the same issue with one 6x 1080Ti Asus Strix 011G. Other rigs working well, just one have 35-37Mhs problem in SMOS with Pill. It shows core clocks <1000 and Power draw about 130W what ever the OC settings

**Comment by odpast** [comment]
I think its fixable because I have 1080Ti Strix and just day before I was able to mine at 48mhs, afterburner crashed and I'm stuck at 34mhs max..
